src/nats.h

```diff
@@ -1276,52 +1271,56 @@ typedef struct jsOptions
+        } PublishAsync; ///< extra options for #js_PublishAsync
 
         struct jsOptionsPullSubscribeAsync
         {
-                // Lifetime of the subscription (completes when any one of the
-                // targets is reached).
-                int64_t                 Timeout; // in milliseconds
-                int                     MaxMessages;
-                int64_t                 MaxBytes;
-
-                // If NoWait is set, the subscription will receive the messages
-                // already stored on the server subject to the limits, but will
-                // not wait for more messages.
-                //
-                // Note that if Timeout is set we would still wait for first
-                // message to become available, even if there are currently any
-                // on the server
+                int64_t                 Timeout;        ///< Auto-unsubsribe after this many milliseconds.
+                int                     MaxMessages;    ///< Auto-unsubscribed after receiving this many messages.
+                int64_t                 MaxBytes;       ///< Auto-unsubscribe after receiving this many bytes.
+
+                /// \brief If NoWait is set, the subscription will receive the
+                /// messages already stored on the server subject to the limits,
+                /// but will not wait for more messages.
+                ///
+                /// \note that if Timeout is set we would still wait for first
+                /// message to become available, even if there are currently any
+                /// on the server
                 bool                    NoWait;
 
-                // Fetch complete handler that receives the exit status code,
-                // the subscription's Complete handler is also invoked, but does
-                // not have the status code.
-                natsFetchCompleteHandler CompleteHandler;
+                /// \brief Fetch complete handler that receives the exit status
+                /// code, the subscription's Complete handler is also invoked,
+                /// but does not have the status code.
+                jsFetchCompleteHandler  CompleteHandler;
                 void                    *CompleteHandlerClosure;
 
-                // Have server sends heartbeats at this interval to help detect
-                // communication failures.
-                int64_t                 Heartbeat; // in milliseconds
-
-                // Options to control automatic Fetch flow control. The number
-                // of messages to ask for in a single request, and if we should
-                // try to fetch ahead, KeepAhead more than we need to finish the
-                // current request. Fetch this many messages ahead of time.
-                //
-                // KeepAhead can not be used in conjunction with MaxBytes or
-                // NoWait.
+                /// \brief Have server sends heartbeats at this interval (in
```

Comment:
NS: Hmm... The `\` vs `@` inconsistency is all over the place in the file; maybe a separate PR (for v4.0??) to make it consistent?